Automattic/wp-calypso
# Enable cancellation flows for Akismet products

## Proposed Changes

* This PR enables cancellation flows for Akismet products from the purchases page

## Testing Instructions

* You will need a sample Akismet purchase to test this flow
* Apply this PR on your local Calypso testing environment
* Navigate to `/me/purchases` and select your Akismet purchase
* From the manage purchase page, you should be able to use the auto-renewal toggle to turn auto-renew off and on again

![Screenshot 2023-03-23 at 4 04 58 PM](https://user-images.githubusercontent.com/18016357/227341588-de1a3a3a-3125-4a1e-8c1f-e2905cb9fddd.png)

* Check that there are no console errors while changing the auto-renew setting
* Now, click on "Cancel Subscription" at the bottom of the management screen

![Screenshot 2023-03-23 at 4 05 03 PM](https://user-images.githubusercontent.com/18016357/227341681-d3898c25-4270-4604-b192-f1b28817a045.png)

* This will take you to the purchase cancellation page

![Screenshot 2023-03-23 at 4 05 10 PM](https://user-images.githubusercontent.com/18016357/227341812-3bf60b13-6120-42cd-8dc3-b138afde48d7.png)

* Click on "Cancel Purchase"

![Screenshot 2023-03-23 at 4 05 18 PM](https://user-images.githubusercontent.com/18016357/227342011-99fe3806-f44f-4cad-8667-bf88e73c4824.png)

* You will be presented with a survey screen. Click "Submit and cancel product" to proceed
* The product should be cancelled successfully. If you are still within the refund window (14 days) the subscription will also be removed from your account.


Regression testing
* Test with both Jetpack and WPCOM purchases to ensure the Auto-renew toggle still works as expected. Particularly, for a disconnected Jetpack site, the toggle should not be present.

## Pre-merge Checklist
